### ISSUE TITLE
[Feature Request] v-calendar add title property to calendar

### ISSUE THREAD

**Issue by ClickerMonkey** [vuetifyjs/vuetify#11182]
### Problem to solve
It's very common to summarize the time span you're looking at (specific Month, Week, Day, etc). Add a simple computed property to the title would make it easier for users and would simplify examples.

### Proposed solution
Add the computed title

**Comment by ClickerMonkey** [comment]
Playground for feature:

<details>

```vue
<template>
  <v-container>
    <div>
      <v-sheet
        tile
        height="54"
        color="grey lighten-3"
        class="d-flex"
      >
        <v-btn
          icon
          class="ma-2"
          @click="$refs.calendar.prev()"
        >
          <v-icon>mdi-chevron-left</v-icon>
        </v-btn>
        <h4 v-if="$refs.calendar" class="pa-4">
          {{ $refs.calendar.title }}
        </h4>
        <v-spacer></v-spacer>
        <v-btn
          icon
          class="ma-2"
          @click="$refs.calendar.next()"
        >
          <v-icon>mdi-chevron-right</v-icon>
        </v-btn>
      </v-sheet>
      <v-sheet height="600">
        <v-calendar
          ref="calendar"
          v-model="value"
          :type="type"
          :events="events"
          :event-color="getEventColor"
          @change="getEvents"
        ></v-calendar>
      </v-sheet>
    </div>
  </v-container>
</template>

<script>
  export default {
    data: () => ({
      type: 'week',
      mode: 'stack',
      value: '',
      events: [],
      colors: ['blue', 'indigo', 'deep-purple', 'cyan', 'green', 'orange', 'grey darken-1'],
      names: ['Meeting', 'Holiday', 'PTO', 'Travel', 'Event', 'Birthday', 'Conference', 'Party'],
    }),
    methods: {
      getEvents ({ start, end }) {
        const events = []

        const min = new Date(`${start.date}T00:00:00`)
        const max = new Date(`${end.date}T23:59:59`)
        const days = (max.getTime() - min.getTime()) / 86400000
        const eventCount = this.rnd(days, days + 20)

        for (let i = 0; i < eventCount; i++) {
          const allDay = this.rnd(0, 3) === 0
          const firstTimestamp = this.rnd(min.getTime(), max.getTime())
          const first = new Date(firstTimestamp - (firstTimestamp % 900000))
          const secondTimestamp = this.rnd(2, allDay ? 288 : 8) * 900000
          const second = new Date(first.getTime() + secondTimestamp)
          const type = this.rnd(0, this.names.length - 1)

          events.push({
            name: this.names[type],
            start: first,
            end: second,
            color: this.colors[this.rnd(0, this.colors.length - 1)],
            timed: !allDay,
          })
        }

        this.events = events
      },
      getEventColor (event) {
        return event.color
      },
      rnd (a, b) {
        return Math.floor((b - a + 1) * Math.random()) + a
      },
    },
  }
</script>
```
</details>

**Comment by johnleider** [comment]
resolved in #11198

If you have any additional questions, please reach out to us in our [Discord community](https://community.vuetifyjs.com).
